test/integration/util/util.go

```diff
@@ -96,8 +96,13 @@ func StartScheduler(ctx context.Context, clientSet clientset.Interface, kubeConf
 		klog.Fatalf("Error creating scheduler: %v", err)
 	}
 
+	logger := klog.FromContext(ctx)
 	informerFactory.Start(ctx.Done())
 	informerFactory.WaitForCacheSync(ctx.Done())
+	if err = sched.WaitForHandlersSync(ctx); err != nil {
+		logger.Error(err, "error occurred during WaitForHandlersSync")
```

Comment:
we shouldn't continue with the test in this case https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#replacing-fatal-calls

Reply:
Ah, I see. Didn't know this before, updated.

Reply:
Should we continue in `app/server.go` when failed waiting for handlers to sync?

Reply:
it seems this has been resolved but I can't see how this stops , logger.Error AFAIK doesn't exit